src/controllers/director.residential.address.controller.ts

```diff
@@ -117,18 +117,22 @@ export const buildValidationErrors = (req: Request): ValidationError[] => {
 
 const formatDirectorResidentialAddress = (officerFiling: OfficerFiling): string => {
   return formatTitleCase(`
+          ${officerFiling.serviceAddress?.premises ? officerFiling.serviceAddress.premises+',' : ""}
           ${officerFiling.serviceAddress?.addressLine1},
-          ${officerFiling.serviceAddress?.addressLine2 ? officerFiling.serviceAddress.addressLine2 : ""}
+          ${officerFiling.serviceAddress?.addressLine2 ? officerFiling.serviceAddress.addressLine2+"," : ""}
```

Comment:
Do we want the additional comma between house number and street name?
Address is displayed as "4, Duncalf Street, Stoke-on-trent, England ST6 3LJ"
But the prototype example in jira shows no comma between House number and street name.